app/protected/template.hbs

```diff
@@ -1,15 +1,8 @@
 <main class="page-main">
   {{#if this.loading}}<span class="loader"></span>{{/if}}
   <div class="page-content page-content--scroll">
-    <SyTopnav @on-logout={{route-action 'invalidateSession'}} />
+    <SyTopnav @on-logout={{this.invalidateSession}} />
 
     {{outlet}}
   </div>
 </main>
-
-<WelcomeModal
-  @visible={{this.visible}}
-  @on-never={{route-action 'neverTour'}}
-  @on-later={{route-action 'laterTour'}}
-  @on-start={{route-action 'startTour'}}
-}}
```

Comment:
Does the tour still work or will we not be showing it anymore?

Reply:
This block should stay in the code. Deleted by accident.

Reply:
Added again.